### ISSUE TITLE
Support using pager for viewing long issues

### ISSUE THREAD

**Issue by aszenz** [go-jira/jira#351]
For viewing issues with many comments, there is often a need to use a pager like less `jira view JT-123 | less`, just wondering if there's a way the cli can use a pager automatically whenever the output is longer than the terminal height or even allow using a pager always in the config.
This option will also be useful for `jira ls` so it could be made for all commands I think

**Comment by ryran** [comment]
Agreed. Would be really nice to have a global option for this (like `--pager`).
